docs/downward_api.md

```diff
@@ -1,28 +1,51 @@
 # Downward API
 
-The downward API allows containers to consume information about the system without coupling to the
+It is sometimes useful for a container to have information about itself, but we
+want to be careful not to over-couple containers to Kubernetes. The downward
+API allows containers to consume information about themselves or the system and
```

Comment:
@thockin What pieces of info that you can get to via the downward API qualify as about the system?